src/pages/Dashboard/ChildComponent.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const ChildComponent = ({ heading, button, onMouseHover }) => {
+  return (
+    <div
+      className="childComponent"
+      style={{ border: "1px solid red", padding: "10px", borderRadius: "10px" }}
```

Comment:
avoid inline styling . implement the styling in styled component.

Reply:
done